07_Bootstrap/Praktikum/landing-page.html

```diff
@@ -0,0 +1,153 @@
+    <link rel="stylesheet" href="./assets/style.css">
+    <title>eStartup</title>
+</head>
+<body> 
+   <!-- navbar -->
+   <nav class="navbar navbar-expand-lg fixed-top border border-2 navbar-light bg-light ">
+    <div class="container">
+      <a class="navbar-brand " href="#"><span class="text-success">e</span>Startup</a>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse justify-content-end  navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav  mb-2  mb-lg-0 text-capitalize">
+          <li class="nav-item">
+            <a class="nav-link " aria-current="page" href="#">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#about">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Features</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">screenshots</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link">team</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link">pricing</a>
+          </li>
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+              Dropdown
+            </a>
+            <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+              <li><a class="dropdown-item" href="#">Action</a></li>
+              <li><a class="dropdown-item" href="#">Another action</a></li>
+              <li><hr class="dropdown-divider"></li>
+              <li><a class="dropdown-item" href="#">Something else here</a></li>
+            </ul>
+          </li>
+        
+        </ul>
+  
+      </div>
+    </div>
+  </nav>
+   <!-- navbar -->
+     <!-- main-content -->
+     <div class="container">
+      <div class="content1 ">
+       <div class="row text-center" >
+           <div class="col-12 text-center text-capitalize mb-5">
+               <h2>simple systeme fordicount</h2>
+               <p>Integer Curcus Augue Ac Curcus</p>
+           </div>
+           <div class="col-lg-4 pb-4 col-md-6 svg">
+               <img src="./assets/svg/cloud.svg" alt="">
+               <p class="fw-bold tittle">introducing whatsapp</p>
+               <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quaerat enim nam iusto unde ut. Officiis libero labore quasi eum totam?</p>
+               <a class="fw-normal text-black" href="">Read More</a>
+           </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)